docs/resources/dms_kafka_instance.md

```diff
@@ -55,21 +95,27 @@ The following arguments are supported:
 * `name` - (Required, String) Specifies the name of the DMS kafka instance. An instance name starts with a letter,
   consists of 4 to 64 characters, and supports only letters, digits, hyphens (-) and underscores (_).
 
-* `product_id` - (Required, String) Specifies a product ID, which includes bandwidth, partition, broker and default
+* `flavor_id` - (Optional, String) Specifies the kafka [flavor ID](https://support.huaweicloud.com/intl/en-us/productdesc-kafka/Kafka-specification.html),
+  e.g. **c6.2u4g.cluster**. This parameter and `product_id` are alternative.
+
+  -> The flavor IDs are not supported in some regions.
+
+* `product_id` - (Optional, String) Specifies a product ID, which includes bandwidth, partition, broker and default
   storage capacity.
 
   -> **NOTE:** Change this to change the bandwidth, partition and broker of the Kafka instances. Please note that the
   broker changes may cause storage capacity changes. So, if you specify the value of `storage_space`, you need to
   manually modify the value of `storage_space` after changing the `product_id`.
 
-* `engine_version` - (Required, String, ForceNew) Specifies the version of the kafka engine. Valid values are "1.1.0"
-  and "2.3.0". Changing this creates a new instance resource.
+* `engine_version` - (Required, String, ForceNew) Specifies the version of the kafka engine. Valid values are **1.1.0**
+  and **2.3.0**. Changing this creates a new instance resource.
 
-* `storage_spec_code` - (Required, String, ForceNew) Specifies the storage I/O specification. Value range:
-  + When bandwidth is 100MB: dms.physical.storage.high or dms.physical.storage.ultra
-  + When bandwidth is 300MB: dms.physical.storage.high or dms.physical.storage.ultra
-  + When bandwidth is 600MB: dms.physical.storage.ultra
-  + When bandwidth is 1,200MB: dms.physical.storage.ultra
+* `storage_spec_code` - (Required, String, ForceNew) Specifies the storage I/O specification.
+  The valid values are as follows:
+  + **dms.physical.storage.high.v2** (old: **dms.physical.storage.high**): Type of the disk that uses high I/O
```

Comment:
it's hard to understand the **old** or **new**

Reply:
done.